dbhydra/dbhydra_core.py

```diff
@@ -1800,45 +1854,92 @@ def concat_with_reset_index_in_second_df(original_df, df):
                 maximal_index=0
             df.index = df.index + maximal_index + 1
             df.reindex(columns=original_df.columns.tolist()) #to ensure that columns get correctly inserted
-            original_df = original_df.append(df)
-            # df=pd.concat([original_df,df])
-            return (original_df)
+            result_df = pd.concat([original_df,df])
+            return (result_df)
 
         df = concat_with_reset_index_in_second_df(original_df, df)
         df["uid"]=df.index
-        df.reindex(columns=["uid"]+df.columns[:-1].tolist()) #uid as a first column
+        df = df.reindex(columns=["uid"]+df.columns[:-1].tolist()) #uid as a first column
         df.reset_index(drop=True,inplace=True)
-        print("INSERTED",df)
-        df.to_excel(self.db1.name + "\\" + self.name + ".xlsx",index=False)
+        df.to_excel(self.db_path, index=False)
 
     def replace_from_df(self, df):
         assert len(df.columns) == len(self.columns)  # +1 because of id column
         #df.drop(df.columns[0], axis=1, inplace=True)
-        df.to_excel(self.db1.name + "\\" + self.name + ".xlsx",index=False)
+        df.to_excel(self.db_path, index=False)
+
+    # def update(self, variable_assign: str, where: Optional[str] = None):
```

Comment:
@Omastto1, @DovaX 
I had to rewrite this method as it would only update a first variable pair of the `variable_assign` arguments. Is there some special use case to this method I'm not aware of?

Reply:
i even think it will update the first variable with the rest of the update statement